2024-08-25 18:45:40,203 INFO c.i.t.p.i.StudentDaoImpl [main] StudentDaoImpl initialized with database URL: jdbc:mysql://localhost:3306/studentapp
2024-08-25 18:45:40,837 INFO c.i.t.p.i.StudentDaoImpl [main] Student Last Name updated successfully for ID 1026
2024-08-25 18:45:40,839 INFO c.i.t.p.i.StudentDaoImpl [main] StudentDaoImpl initialized with database URL: jdbc:mysql://localhost:3306/studentapp
2024-08-25 18:45:40,854 INFO c.i.t.p.i.StudentDaoImpl [main] Student First Name updated successfully for ID 1026
2024-08-25 18:45:40,855 INFO c.i.t.p.i.StudentDaoImpl [main] StudentDaoImpl initialized with database URL: jdbc:mysql://localhost:3306/studentapp
2024-08-25 18:45:40,890 ERROR c.i.t.p.i.StudentDaoImpl [main] Failed to insert student data
java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '1026' for key 'students.PRIMARY'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:117) ~[mysql-connector-java-8.0.29.jar:8.0.29]
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122) ~[mysql-connector-java-8.0.29.jar:8.0.29]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:916) ~[mysql-connector-java-8.0.29.jar:8.0.29]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1061) ~[mysql-connector-java-8.0.29.jar:8.0.29]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1009) ~[mysql-connector-java-8.0.29.jar:8.0.29]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1320) ~[mysql-connector-java-8.0.29.jar:8.0.29]
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:994) ~[mysql-connector-java-8.0.29.jar:8.0.29]
	at com.indus.training.persist.impl.StudentDaoImpl.insertStudent(StudentDaoImpl.java:64) ~[classes/:?]
	at com.indus.training.persist.dao.TestStudentDaoImpl.testInsertStudent(TestStudentDaoImpl.java:27) ~[test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_202]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_202]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_202]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_202]
	at junit.framework.TestCase.runTest(TestCase.java:177) ~[junit-4.13.2.jar:4.13.2]
	at junit.framework.TestCase.runBare(TestCase.java:142) ~[junit-4.13.2.jar:4.13.2]
	at junit.framework.TestResult$1.protect(TestResult.java:122) ~[junit-4.13.2.jar:4.13.2]
	at junit.framework.TestResult.runProtected(TestResult.java:142) ~[junit-4.13.2.jar:4.13.2]
	at junit.framework.TestResult.run(TestResult.java:125) ~[junit-4.13.2.jar:4.13.2]
	at junit.framework.TestCase.run(TestCase.java:130) ~[junit-4.13.2.jar:4.13.2]
	at junit.framework.TestSuite.runTest(TestSuite.java:241) ~[junit-4.13.2.jar:4.13.2]
	at junit.framework.TestSuite.run(TestSuite.java:236) ~[junit-4.13.2.jar:4.13.2]
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:90) ~[junit-4.13.2.jar:4.13.2]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93) ~[.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40) ~[.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529) ~[.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:757) ~[.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452) ~[.cp/:?]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210) ~[.cp/:?]
2024-08-25 18:45:40,896 INFO c.i.t.p.i.StudentDaoImpl [main] StudentDaoImpl initialized with database URL: jdbc:mysql://localhost:3306/studentapp
2024-08-25 18:45:40,898 ERROR c.i.t.p.d.TestStudentDaoImpl [main] I am Logger
2024-08-25 18:45:40,918 INFO c.i.t.p.i.StudentDaoImpl [main] Student data successfully fetched for ID 1026
